app/api/private-cloud/products/[licencePlate]/requests/route.ts

```diff
@@ -16,6 +17,7 @@ const queryParamSchema = z.object({
 
 const apiHandler = createApiHandler({
   roles: ['user'],
+  permissions: [PermissionsEnum.ViewAllPrivateCloudProductsHistory],
```

Comment:
please consider removing this permission; this api endpoint is for all users and I will get data what they can get only based on the security layer.